source/mainnet/net/guides/overview-desktop.rst

```diff
@@ -7,19 +7,19 @@ Set up the Desktop Wallet
 
 The Concordium Desktop Wallet is a digital wallet that enables you to create and manage your Concordium identities, credentials, and accounts, and to create transactions such as sending CCD, adding a baker, and exporting and importing account information.
 
-The Desktop Wallet also supports storing of your private keys on a hardware device. Currently, only the Ledger Nano S is supported.
+The Desktop Wallet also supports storing of your private keys on a hardware device. Currently, the Ledger Nano S and Ledger Nano S Plus are supported.
 
-This topic contains an overview of each task you need to complete to set up and start using the Desktop Wallet. The guide assumes that you'll be using the Desktop Wallet in combination with a Ledger Nano S hardware device to generate and store keys and sign transactions.
+This topic contains an overview of each task you need to complete to set up and start using the Desktop Wallet. The guide assumes that you'll be using the Desktop Wallet in combination with a Ledger device to generate and store keys and sign transactions.
 
 Step 1: Set up the node
 =======================
 
 The Desktop Wallet must be connected to a running **trusted** node on the Concordium blockchain, so you'll have to run a node. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and submit transactions. You can run a node using :ref:`Windows<run-node-windows>`, :ref:`macOS <run-node-macos>`, and :ref:`with Docker <run-a-node>` or :ref:`a Debian package <run-node-ubuntu>` on Linux. You can also have a third-party provider run a node for you.
 
-Step 2: Set up the Ledger Nano S
+Step 2: Set up the Ledger device
 ================================
 
-The Desktop Wallet requires that you store your keys on a Ledger Nanos S hardware device. This is to ensure that your private account keys are kept secure. To be able to use the Ledger Nano S with the Desktop Wallet, you must install the Concordium Ledger App on the hardware wallet. See :ref:`Install the Ledger App guide<install-ledger-app>`.
+The Desktop Wallet requires that you store your keys on a Ledger device. This is to ensure that your private account keys are kept secure. To be able to use the Ledger Nano S with the Desktop Wallet, you must install the Concordium Ledger App on the hardware wallet. See :ref:`Install the Ledger App guide<install-ledger-app>`.
```

Comment:
To be able to use the Ledger Nano S -> has to also mention Ledger Nano S Plus, or just say Ledger device.